astro/src/content/quickstarts/quickstart-ruby-on-rails-api.mdx

```diff
@@ -0,0 +1,351 @@
+
+<Aside type="note">
+If you ever want to reset the FusionAuth system, delete the volumes created by docker-compose by executing `docker-compose down -v`, then re-run `docker-compose up -d`.
+</Aside>
+
+FusionAuth will be initially configured with these settings:
+
+* Your client Id is: `e9fdb985-9173-4e01-9d73-ac2d60d1dc8e`
+* Your client secret is: `super-secret-secret-that-should-be-regenerated-for-production`
+* Your example teller username is `teller@example.com` and your password is `password`. They will have the role of `teller`.
+* Your example customer username is `customer@example.com` and your password is `password`. They will have the role of `customer`.
+* Your admin username is `admin@example.com` and your password is `password`.
+* Your fusionAuthBaseUrl is 'http://localhost:9011/'
+
+You can log into the [FusionAuth admin UI](http://localhost:9011/admin) and look around if you want, but with Docker/Kickstart you don't need to.
+
+## Create Your Ruby on Rails Resource Server Application
+
+Now you are going to create a Ruby on Rails API application. While this section builds a simple API, you can use the same configuration to integrate an existing API with FusionAuth.
+
+We are going to be building an API backend for a banking application called ChangeBank. This API will have two endpoints:
+- `make-change`: This endpoint will allow you to call GET with a `total` amount and receive a response indicating how many nickels and pennies are needed to make change. Valid roles are `customer` and `teller`.
+- `panic`: Tellers may call this endpoint to call the police in case of an incident. The only valid role is `teller`.
+
+Both endpoints will be protected such that a valid JSON web token (JWT) will be required in the `Authorization` header in order to be accessed. Additionally, the JWT must have a `roles` claim containing the appropriate role to use the endpoint.
+
+<Aside type="note">
+If you simply want to run the application , there is a completed version in the 'complete-application' directory. You can use the following commands to get it up and running if you do not want to create your own.
+
+```shell
+cd complete-application
+bundle install
+bundle e rails s -p 4001
+```
+
+You can then follow the instructions in the [Run the API](#run-the-api) section started at Get a Token.
+</Aside>
+
+### Initialize The Application
+
+Initialize the Ruby on Rails application using the following:
+
+```
+rails new your-application --api
+cd your-application
+```
+
+Once this is complete you will see a new directory called `your-application` with several sub directories.
+
+You need to update the `Gemfile` to include two new dependencies:
+
+```shell
+gem 'rack-jwt', git: 'https://github.com/FusionAuth/rack-jwt'
+gem 'dotenv-rails'
+```
```

Comment:
This does not work for me, I get an Error.
![Screenshot 2023-09-15 at 2 39 10 PM](https://github.com/FusionAuth/fusionauth-site/assets/137117976/a98722c1-ea63-4556-91fe-becd4f88726c)